backend/run_server.sh

```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=app
```

Comment:
There is already satnogs-web.py script. If you don't like and want to replace it - fine. We should have one run script, not two.

Reply:
I deleted my version.